edb/server/defines.py

```diff
@@ -27,7 +27,7 @@
 EDGEDB_VISIBLE_METADATA_PREFIX = r'EdgeDB metadata follows, do not modify.\n'
 
 # Increment this whenever the database layout or stdlib changes.
-EDGEDB_CATALOG_VERSION = 20191128_00_00
+EDGEDB_CATALOG_VERSION = 20191205_00_00
```

Comment:
Change to this, otherwise you will have a conflict with master (your other PR just got merged)

```suggestion
EDGEDB_CATALOG_VERSION = 20191205_01_00
```

Reply:
Ah, here we are, committed to the wrong branch :(